src/components/fetch-wrapper.jsx

```diff
@@ -7,7 +7,7 @@ import Loading from './loading';
 
 function withFetching(WrappedComponent, props) {
   return class extends Component {
-    constructor(props) {
+    constructor(props) { // eslint-disable-line no-shadow
```

Comment:
curious what is this for?

Reply:
I noticed this component has two variables called `props`, inside the params for `withFetching` and here in `constructor`. This was previously causing a lint error. I added this comment to minimally disrupt this logic.